source/code/include/playfab/PlayFabAuth.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !defined(__cplusplus)
+#error C++11 required
+#endif
+
+#pragma once
+
+#include <httpClient/pal.h>
+#include <httpClient/async.h>
+
+extern "C"
+{
+
+/// <summary>
+/// Handle to an authenticated entity (Player, Server, etc.). Contains the auth info needed to make PlayFab service
+/// calls. When no longer needed, auth handle must be closed with PlayFabAuthenticatedEntityCloseHandle.

```

Comment:
>PlayFabAuthenticatedEntityCloseHandle [](start = 66, length = 37)

PlayFabAuthContextCloseHandle (this function was renamed and forgot to update comments)